workshops/templates/workshops/person.html

```diff
@@ -73,6 +73,11 @@
       </li>
       {% endfor %}
     </ul>
+    <ul>
+      <li> Has taught {{ person.task_set.instructors.count }} time{{ person.task_set.instructors.count|pluralize }}</li>
+      <li> Has helped {{ person.task_set.helpers.count }} time{{ person.task_set.helpers.count|pluralize }}</li>
+      <li> Has learned {{ person.task_set.learners.count }} time{{ person.task_set.learners.count|pluralize }}</li>
```

Comment:
However, annotations are defined on querysets.
Do you want me to write a query with the `pk` parameter?